src/test/java/com/epam/atm/selenium/CreateNewCasePOTest.java

```diff
@@ -4,7 +4,7 @@
 import org.testng.annotations.Test;
 import pageobject.*;
 
-public class CreateNewCasePOTest extends BaseTest {
+public class CreateNewCasePOTest extends BaseLocalTest {
```

Comment:
Remove duplication. Login password already in base class

Reply:
done